app/packages/operators/src/components/OperatorExecutionTrigger/index.tsx

```diff
@@ -46,17 +54,40 @@ export const OperatorExecutionTrigger = ({
   onClick?: () => void;
   onSuccess?: ExecutionCallback;
   onError?: ExecutionErrorCallback;
-  prompt?: boolean;
   executionParams?: object;
   executorOptions?: OperatorExecutorOptions;
-  executionOptions?: OperatorExecutionOption[];
   onOptionSelected?: (option: OperatorExecutionOption) => void;
   disabled?: boolean;
 }) => {
   const [isMenuOpen, setIsMenuOpen] = useState(false);
   // Anchor to use for context menu
   const containerRef = useRef(null);
 
+  // Pass onSuccess and onError through to the operator executor.
```

Comment:
Hmm this is only partially reverting... ugh. Going to close this and start over.